erdblick_app/app/map.panel.component.ts

```diff
@@ -215,18 +222,18 @@ export class MapPanelComponent {
 
     focus(tileId: bigint, event: any) {
         event.stopPropagation();
-        if (this.mapService.mapModel.getValue() && this.mapService.coreLib !== undefined) {
-            this.mapService.mapModel.getValue()!.zoomToWgs84PositionTopic.next(
-                this.mapService.coreLib.getTilePosition(BigInt(tileId))
+        if (this.coreService.coreLib !== undefined) {
```

Comment:
These checks should not be necessary anymore. If they are, maybe add a shorthand lib() method to coreService.